src/main/java/de/zalando/aruha/nakadi/config/RepositoriesConfig.java

```diff
@@ -26,8 +32,19 @@
     @Autowired
     private JdbcTemplate jdbcTemplate;
 
+    @Autowired
+    private ZookeeperConfig zookeeperConfig;
+
     @Bean
-    public EventTypeRepository eventTypeRepository() {
-        return new EventTypeDbRepository(jdbcTemplate, jsonConfig.jacksonObjectMapper());
+    public EventTypeRepository eventTypeRepository() throws Exception {
+        final EventTypeRepository dbRepo = new EventTypeDbRepository(jdbcTemplate, jsonConfig.jacksonObjectMapper());
+
+        final String connectString = zookeeperConfig.zooKeeperHolder().get().getZookeeperClient().getCurrentConnectionString();
+        final RetryPolicy retryPolicy = new ExponentialBackoffRetry(1000, 3);
+        final CuratorFramework client = CuratorFrameworkFactory.newClient(connectString, retryPolicy);
```

Comment:
why do you create a seconds CuratorFramework here? 
We already have one in https://github.com/zalando/nakadi/blob/nakadi-jvm/src/main/java/de/zalando/aruha/nakadi/repository/zookeeper/ZooKeeperHolder.java#L29